PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -45,11 +45,11 @@
     <FilesToCopy Include="$(SourcesRoot)\V0_9UpgradeMessageExe\bin\$(Configuration)\SonarQube.MSBuild.PreProcessor.exe" />
     <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.PreProcessor\bin\$(Configuration)\MSBuild.SonarQube.Internal.PreProcess.exe" />
     <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.PreProcessor\bin\$(Configuration)\SupportedBootstrapperVersions.xml" />
-    <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.PreProcessor\bin\$(Configuration)\Newtonsoft.Json.dll" />
```

Comment:
The situation with the Newtonsoft dll is now slightly more complicated because of the move to NuGet.exe.
NuGet.exe embeds a complete copy of a version of the Newtonsoft code (which is licensed under the MIT license). This means that the pre-process can't reference both Newtonsoft.Json.dll and NuGet.exe at the same time, so it references NuGet.exe.
However, the SonarRunner.Shim also needs to process JSON (to fix faulty SARIF files), and it does not use NuGet so it still references the Newtonsoft dll.
Consequently, we ship both, and this project now references the Newtonsoft dll used in the SonarRunner shim.